app/controllers/programs_controller.rb

```diff
@@ -46,8 +47,7 @@ def program_data
 
   # POST /programs or /programs.json
   def create
-    
-    @program = Program.new(program_params.except(:instructor_attributes))
+    @program = Program.new(program_params.except(:instructor_attributes, :copy_from_id))
```

Comment:
do we need to add **:copy_from_id** in the list of trusted params on line 134 ?

Reply:
**Update** - Rita renamed _:copy_from_id_ to _duplicate_program_id_ and also added _duplicate_program_id to the list of trusted params

Reply:
@anantaUM I added the variable (:duplicate_program_id) to the list of params. But In the documentation, it says that this list is "to set the parameter as permitted and limit which attributes should be allowed for mass updating". duplicate_program_id is not used in the update, and the script works without adding that variable to the list of params, so I am not sure if it makes any difference if that variable is in the list or not.

Reply:
@britaumich - Ok. Let us confirm with @rsmoke or @dschmura so that we can handle similar scenarios in future.